src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -120,4 +134,97 @@ public void visit(AssignmentExpr assignmentExpr) {
   private void space() {
     buffer.append(SPACE);
   }
+
+  /** =============================== STATEMENT =============================== */
+  public String visit(LineComment linecomment){
+      // Split comments by new line and add `//` to each line.
+      String[] sourceStrings = linecomment.comment().split("\\r?\\n");
+      for (int i = 0; i < sourceStrings.length; i++) {
+        sourceStrings[i] = "// " + sourceStrings[i];
+      }
+      String formattedSource =format(String.join("\n", sourceStrings));
+      return formattedSource;
+  }
+
+  public String visit(BlockComment blockComment){
+    // Split comments by new line and embrace the comment block with `/** */`.
+    String sourceString = blockComment.comment();
+    String formattedSource = format("/** " + sourceString + " */");
```

Comment:
Consider using `String.format` instead of concatenation.

Reply:
done. thanks!